### ISSUE TITLE
Add support for OpenType Fonts

### ISSUE THREAD

**Issue by richso** [TIBCOSoftware/jaspersoft-studio-ce#64]
When will supporting for OpenType Fonts be added?

**Comment by gt78** [comment]
Hi Richso, thank you for your message. The support for OpentType Fonts in Studio will be supported as soon it will be supported by JasperReports Library. So I will point my colleagues working on JR to your request.
Good you were able to use OTF fonts, but we may need to verify that this type of fonts don't cause side effects in all exporters.
As a fall back solution, you can always convert OTF to TTF.

Regards,
Giulio

**Comment by richso** [comment]
FYI, the cheating method work for iTEXT7 using the iText7 support library (https://github.com/Jaspersoft/jasperreports-pdf-lib7), i.e. generate PDF which can be opened by Adobe Reader without problem but this iText7 supporting library have HTML markup layout problems than the original Classic PDF producer.

**Comment by teodord** [comment]
The Lobster Two font in our fonts sample in the JRL project is an OTF file:
https://github.com/TIBCOSoftware/jasperreports/tree/master/jasperreports/demo/samples/fonts/src/lobstertwo

**Comment by gt78** [comment]
Good point Teodor... then this becomes a Request for Enhancement for JSS...

**Comment by richso** [comment]
Hi Giulio, 

I just tested with your suggestion with the CJK OTF (https://github.com/adobe-fonts/source-han-sans), I got my same error when opening with Adobe Reader and the Chinese words displayed as blanks.

![adobe-error](https://user-images.githubusercontent.com/5850283/233763979-2630596d-b5ae-4e0b-b4b0-d1a1604d5aa6.png)
![adobe2](https://user-images.githubusercontent.com/5850283/233764026-09186ea3-9299-4b5d-adeb-94eb34681c12.png)

This is the normal display by browser:
![adobe-normal](https://user-images.githubusercontent.com/5850283/233764076-e4966ebb-763f-4287-9f5d-e24f008fc344.png)
